.github/workflows/code-quality.yml

```diff
@@ -33,23 +33,40 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: 3.8
-        
-      - name: Install Black and clang-format
+
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+
+      - name: Install and run isort
+        run: |
+          pip install isort
+          isort src tests scripts
```

Comment:
@bramstoeller 

why not just `isort .`?

Same for `black` and `mypy`.

Reply:
black also tries to format jupyter notebooks, which is not desirable imho. And then I decided to explicitly list the directories that we'd like to check.

Reply:
black only check jupyter notebooks if you install black with `pip install black[jupyter]`. If you install the default black, it will ignore jupyter.